README.md

```diff
@@ -8,7 +8,7 @@
 - [The Stack](#the-stack)
   - [Infra](#infra)
   - [Arc](#arc)
-  - [Arc.js](#arcjs)
+  - [Client](#client)
```

Comment:
where does subgraph is getting in ?
the client is a layer on top the subgraph

Reply:
So I briefly talked about subgraph in the Client section. Should I remove it or are you suggesting to mention that as part of Stack?

Reply:
I think this should be mentioned as a part of the stack, though you're right it's strongly connected to the client.

Reply:
subgraph (our indexing caching layer) is part of our stack.
one can build its own client on top of the graph service .

Reply:
Gotcha 👍🏻